apps/chat/src/utils/app/import-export.ts

```diff
@@ -446,3 +449,29 @@ export const updateAttachment = ({
   };
   return updatedAttachment;
 };
+
+export const getCancelImportAction = (featureType: FeatureType) => {
+  switch (featureType) {
+    case FeatureType.Chat:
+      return ImportExportActions.cancelConversationsImportDuplicated;
+    case FeatureType.Prompt:
+      return ImportExportActions.cancelPromptsImportDuplicated;
+    //TODO implement case FeatureType.File
+
+    default:
+      return ImportExportActions.cancelConversationsImportDuplicated;
+  }
+};
+
+export const getFolderType = (featureType: FeatureType) => {
```

Comment:
move it in `mappers.ts`

Reply:
Thank you for the suggestion!
Method moved to the mappers.ts